group_vars/matrix_servers

```diff
@@ -2337,6 +2337,9 @@ matrix_synapse_redis_password: "{{ matrix_redis_connection_password if matrix_re
 #
 ######################################################################
 
+matrix_synapse_container_extra_arguments: "{{ matrix_homeserver_container_extra_arguments }}"
+matrix_synapse_app_service_config_files: "{{ matrix_homeserver_container_app_service_config_files }}"
```

Comment:
These should be a few lines above, so that they are within the `matrix-synapse` section.

Reply:
Oh, I see. FIxed.